### PR TITLE
Allow loadbalancer timeout to be configureable

### PR DESCRIPTION
**Pull request by jvisker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,6 +124,7 @@ module "my_app" {
...
 | lb_logging_bucket_name            | string                                      | Required if `lb_logging_enabled` is true. A bucket to store the logs in with an a [load balancer access policy](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-access-logs.html#attach-bucket-policy) attached.                                                                                                                                                              |                                                                                        |
+| lb_idle_timeout                   | number                                      | The time in seconds that the connection is allowed to be idle.                                                                                                                                                                                                                                                                                                                                       |
```
Comment (yoshutch) [inline README.md]: is there a default timeout? should we say what the AWS default timeout is here?

**Review by GaryGSC — COMMENTED** [review]
I'm suddenly annoyed that we're inconsistent with our prefixes. Most variables here use `alb_`, but then we also have some for `lb_logging_`.

Anyway, two nits:
- I'd call this a minor version update
- `lb_` -> `alb_`

**Comment by jvisker** [comment]
@GaryGSC I switched that. I didn't even notice we had two different prefixes going on.

File: README.md (hunk 1 above)
Comment (jvisker) [reply README.md]: Done

**Review by GaryGSC — APPROVED** [review]

**Pull request merged**